runtime/compiler/control/J9Options.cpp

```diff
@@ -1599,6 +1604,31 @@ void J9::Options::preProcessJniAccelerator(J9JavaVM *vm)
       }
    }
 
+uint64_t getCodeCacheMaxPercentageOfAvailableMemory(J9JavaVM *vm)
+   {
+   PORT_ACCESS_FROM_JAVAVM(vm);
+   OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+
+   double codeCacheTotalPercentage = CODECACHE_DEFAULT_MAXRAMPERCENTAGE;
+   char *xxccPercentOption = J9::Options::_externalOptionStrings[J9::ExternalOptions::XXcodecachetotalMaxRAMPercentage];
+   int32_t XXcodeCacheTotalPercentArg = FIND_ARG_IN_VMARGS(STARTSWITH_MATCH, xxccPercentOption, 0);
+   if (XXcodeCacheTotalPercentArg >= 0)
+      {
+      IDATA returnCode = GET_DOUBLE_VALUE(XXcodeCacheTotalPercentArg, xxccPercentOption, codeCacheTotalPercentage);
+      if (OPTION_OK == returnCode)
+         {
+         if (!(codeCacheTotalPercentage > 1.0 && codeCacheTotalPercentage <= 100.0))
```

Comment:
`codeCacheTotalPercentage > 1.0`  this should be `codeCacheTotalPercentage >= 1.0`